src/components/InputNumber/InputNumber.spec.tsx

```diff
@@ -112,4 +112,24 @@ describe('InputNumber', () => {
 		const unit = container.querySelector('span');
 		expect(unit).toHaveTextContent('kg');
 	});
+
+	it('should call setSelectionRange with (0, 0) on blur', () => {
```

Comment:
Je mettrais un nom de test plus parlant tel que `"should display the input value from the start"`, pour parler de la fonctionnalité plutôt que la solution technique.